semantic-conventions/src/tests/data/yaml/scope.yaml

```diff
@@ -0,0 +1,12 @@
+groups:
+  - id: scope
+    prefix: scope
+    brief: >
+      Instrumentation Scope attributes
+    attributes:
```

Comment:
If you don't set any `type: scope` here it will just fall back to the default span (but still work). This way the test doesn't cover your newly introduced type at all, however.